### DISCUSSION TITLE
Additional badge idea (not important / just for fun)

### DISCUSSION THREAD

**Discussion by RomainLLC** [community/community#45454] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello,

For geeks and 7/7 devs around (and i'm pretty sure many could be concerned) could there be a "Committed on Christmas day" achievement 😄?

[Related tweet with badge idea](https://twitter.com/Romain_LLC/status/1618335551383572480)

![image](https://user-images.githubusercontent.com/33468314/214906006-e26161f9-2741-46c7-9266-f9681e504432.png)

**Comment by jge162** [comment]
thats a cool idea, seasonal badges...
